config/config.go

```diff
@@ -154,6 +155,7 @@ func DefaultTestConfig() Config {
 func defaultBaseConfig() BaseConfig {
 	return BaseConfig{
 		DataDirParent:       defaultDataDir,
+		FileLock:            "/tmp/spacemesh.lock",
```

Comment:
I think we need to make it platform independent?

Reply:
yeah i realized that it is probablyy no ok on windows. will look up what works there

Reply:
actually there was os.TempDir